client/src/components/forms/MailingList/slides/Feedback.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { Dispatch, SetStateAction } from 'react';
+import /*type*/ { WithStyles, Theme } from '@material-ui/core/styles';
+import { withStyles, createStyles } from '@material-ui/core/styles';
+import { Typography, FormGroup, FormControlLabel, Checkbox, TextField } from '@material-ui/core';
+import { EVENT } from '../types';
+
+const styles = (theme: Theme) => createStyles({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+  textContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    marginBottom: theme.spacing(2)
+  },
+  text: {
+    color: theme.palette.text.primary
+  },
+  eventsContainer: {
+    marginBottom: theme.spacing(3)
+  },
+  feedback: {
+    width: '90%'
+  }
+});
+
+interface Props extends WithStyles<typeof styles> {
+  events: EVENT[];
+  setEvents: Dispatch<SetStateAction<EVENT[]>>;
+  feedback: string;
+  setFeedback: Dispatch<SetStateAction<string>>;
+}
+
+function Feedback({ classes, events, setEvents, feedback, setFeedback }: Props) {
+  const toggleEvent = (event: EVENT) => {
+    const index = events.indexOf(event);
+    if(index > -1) {
+      events.splice(index, 1);
+    } else {
+      events.push(event);
+    }
+    setEvents(events);
+  }
+
+  return (
+    <div className={classes.container}>
+      <div className={classes.textContainer}>
+        <Typography variant="h4" className={classes.text}>
```

Comment:
I think we can save defining a `text` class by doing `color="textPrimary"`.
https://material-ui.com/api/typography/#props

Edit: Nvm now seeing you used it lower down.